aws/generate.go

```diff
@@ -0,0 +1,117 @@
+// +build ignore
```

Comment:
Should this be codegen? See the `go:generate` command in `doc.go`.

Reply:
not i this case the `generate.go` file is supposed to use a magical `+build ignore` tag that will only be enabled when `go generate` is called, which will implicitly add the tag.

Reply:
will update docs for `go:generate` command to clarify this behavior with `ignore`